etc/pipeline-build/go/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM golang:latest
 
 WORKDIR /app
-ADD build.sh .
-ADD run.sh .
+ADD etc/pipeline-build/go/build.sh .
+ADD etc/pipeline-build/go/run.sh .
```

Comment:
wouldn't a better change be to make whatever does `docker build` of this first `cd` into `etc/pipeline-build/go`? So that anyone can build this docker file from that directory.